kolibri2zim/scraper.py

```diff
@@ -75,20 +79,34 @@ def get_kolibri_url_for(file_id: str, ext: str):
     return f"{STUDIO_URL}/content/storage/{remote_path}", fname
 
 
-def read_from_zip(ark, member, as_text: Optional[bool] = True):
-    data = ark.open(member).read()
-    return data.decode("utf-8") if as_text else data
+def read_from_zip_as_bytes(ark, member):
```

Comment:
I personnaly find it less confusing to only specify the special case one and use the obvious name for the obvious usage. 

```py
def read_from_zip(ark, member: str) -> bytes:
    ...

def read_as_text_from_zip(ark, member: str) -> str:
    ...
```

Reply:
I remove the `read_as_text_from_zip`, it makes no sense to assume globally the encoding + this function is used only once anyway.